`latest` is not pushed to docker hub.

```
13.1.2-ce.0: digest: sha256:8d49b888cbfd738af2f8391072d22562be04d8f53ff65dd9528574b8c369ad3c size: 1571
The push refers to repository [docker.io/***/gitlab-ce]
tag does not exist: ***/gitlab-ce:latest
##[error]Process completed with exit code 1.
```

--- a/.github/workflows/deploy-image.yml
+++ b/.github/workflows/deploy-image.yml
@@ -17,6 +17,7 @@ jobs:
           export RELEASE=$(grep "RELEASE_VERSION=" Dockerfile | sed 's|^.*=||g' |awk '{print $1}')
           docker pull polinux/gitlab-ce:stage
           docker tag polinux/gitlab-ce:stage polinux/gitlab-ce:${RELEASE}
+          docker tag polinux/gitlab-ce:stage polinux/gitlab-ce:latest
           docker login  -u ${{ secrets.DOCKER_HUB_USER }} -p ${{ secrets.DOCKER_HUB_PASS }}
           docker push polinux/gitlab-ce:${RELEASE}
           docker push polinux/gitlab-ce:latest

